_posts/2015-05-08-rustful-0-1-0.md

```diff
@@ -323,7 +323,7 @@ are demonstrated.
 
 ## So, What Happens Next?
 
-This is only the beginning, and there are still a things to do (especially
+This is only the beginning, and there are still a lot of things to do (especially
```

Comment:
I wanted to change it to "and there are still things to do", but it looks like I left a rouge "a" in there. I'm actually not sure what sounds best...